server/src/routes/user.ts

```diff
@@ -0,0 +1,11 @@
+import {Router} from "express";
+import {getUserData,deleteUser, createUser,updateUser} from "../controller/user";
+
+const router = Router();
+
+router.get("/",getUserData);
+router.delete('/userDelete',deleteUser);
+router.post('/userCreate',createUser);
+router.put('/updateUser',updateUser)
```

Comment:
route names should not have method name in it
Like router.delete('/user',deleteUser);

Reply:
Google more for the REST API naming conventions